ts/test/session/sending/PendingMessageCache_test.ts

```diff
@@ -3,44 +3,40 @@ import * as sinon from 'sinon';
 import uuid from 'uuid';
 
 import { ChatMessage } from '../../../session/messages/outgoing';
-import { MessageUtils, PubKey } from '../../../session/utils';
+import * as MessageUtils from '../../../session/utils';
+
+import { TestUtils } from '../../../test/test-utils';
 import { PendingMessageCache } from '../../../session/sending/PendingMessageCache';
 import { RawMessage } from '../../../session/types/RawMessage';
+import { PubKey } from '../../../session/types';
+import * as Data from '../../../../js/modules/data';
 
 describe('PendingMessageCache', () => {
   const sandbox = sinon.createSandbox();
   let pendingMessageCacheStub: PendingMessageCache;
 
   // tslint:disable-next-line: promise-function-async
   const wrapInPromise = (value: any) =>
-    new Promise(r => {
-      r(value);
-    });
-
-  const generateUniqueMessage = (): ChatMessage => {
-    return new ChatMessage({
-      body: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-      identifier: uuid(),
-      timestamp: Date.now(),
-      attachments: undefined,
-      quote: undefined,
-      expireTimer: undefined,
-      lokiProfile: undefined,
-      preview: undefined,
+    new Promise(resolve => {
+      resolve(value);
     });
-  };
 
   beforeEach(async () => {
-    const mockStorageObject = wrapInPromise([] as Array<RawMessage>);
-    const voidPromise = wrapInPromise(undefined);
 
-    // Stub out methods which touch the database.
-    sandbox
-      .stub(PendingMessageCache.prototype, 'getFromStorage')
-      .returns(mockStorageObject);
-    sandbox
-      .stub(PendingMessageCache.prototype, 'syncCacheWithDB')
-      .returns(voidPromise);
+    // Stub out methods which touch the database
+    const storageID = 'pendingMessages';
+    let data: Data.StorageItem = {
+      id: storageID,
+      value: '',
+    };
+
+    TestUtils.stubData('getItemById').withArgs('pendingMessages').callsFake(async () => {
+      return wrapInPromise(data);
+    });
+
+    TestUtils.stubData('createOrUpdateItem').callsFake((item: Data.StorageItem) => {
+      data = item;
```

Comment:
Just a minor thing on this
```suggestion
if (item.id === storageID) { data = item; }
```